x-pack/plugins/alerting/server/routes/rule/apis/bulk_untrack/bulk_untrack_alert_route.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { IRouter } from '@kbn/core/server';
+import {
+  BulkUntrackRequestBodyV1,
+  bulkUntrackBodySchemaV1,
+} from '../../../../../common/routes/rule/apis/bulk_untrack';
+import { transformRequestBodyToApplicationV1 } from './transforms';
+import { ILicenseState, RuleTypeDisabledError } from '../../../../lib';
+import { verifyAccessAndContext } from '../../../lib';
+import { AlertingRequestHandlerContext, INTERNAL_BASE_ALERTING_API_PATH } from '../../../../types';
+
+export const bulkUntrackAlertRoute = (
+  router: IRouter<AlertingRequestHandlerContext>,
+  licenseState: ILicenseState
+) => {
+  router.post(
+    {
+      path: `${INTERNAL_BASE_ALERTING_API_PATH}/rules/_bulk_untrack`,
```

Comment:
I think we should not use rules but alerts here
```suggestion
      path: `${INTERNAL_BASE_ALERTING_API_PATH}/alerts/_bulk_untrack`,
```